resources/generate_book.py

```diff
@@ -462,6 +495,48 @@ def make_book(rm: ResourceManager, i18n: I18n, local_instance: bool = False):
+            entity('tfc:cow', 'A cow.', 'Cow', scale=1),
+            text('$(thing)Cows$() spawn in most climates, between $(l:the_world/climate#temperature)temperature$() -10 and 35°C, and with a $(l:the_world/climate#rainfall)rainfall$() of at least 250mm. They are $(thing)dairy animals$(). They only eat $(item)grains$(), which may be rotten. They can have 1-2 children, are pregnant for 58 days, and reach adulthood in 192 days. They can have children 13 times, if they are never milked, or be milked 128 times, if they are never bred. They produce milk every day.'),
+            entity('tfc:goat', 'A goat.', 'Goat', scale=1),
+            text('$(thing)Goats$() spawn in moderate climates, $(l:the_world/climate#temperature)temperature$() between -12 and 25°C, with a $(l:the_world/climate#rainfall)rainfall$() of at least 300mm. They are $(thing)dairy animals$(). They eat $(item)grains$(), $(item)fruits$(), and $(item)vegetables$(), which may be rotten. They can have 1-2 children, are pregnant for 32 days, and reach adulthood in 96 days. They can have children 6 times if they are never milked, or be milked 60 times if they are never bred. They produce milk every 3 days.'),
+            entity('tfc:yak', 'A yak.', 'Yak', scale=1),
+            text('$(thing)Yaks$() spawn in cold climates, with $(l:the_world/climate#temperature)temperature$() of at most -11°C and at least 100mm of $(l:the_world/climate#rainfall)rainfall$(). They are $(thing)dairy animals$(). They eat only fresh $(item)grains$(). They always have 1 child, are pregnant for 64 days, and reach adulthood in 180 days. They can have children 23 times, if they are never milked, or be milked 230 times, if they are never bred. They produce milk once a day.'),
+            entity('tfc:alpaca', 'An alpaca.', 'Alpaca', scale=1),
+            text('$(thing)Alpacas$() spawn in colder climates, $(l:the_world/climate#temperature)temperature$() between -8 and 20°C, with at least 250mm of $(l:the_world/climate#rainfall)rainfall$(). They are $(thing)wooly animals$(). They eat $(item)grains$() and $(item)fruits$(). They have 1-2 children, are pregnant for 36 days, and reach adulthood in 98 days. They can have children 13 times, if they are never sheared, or be sheared 128 times, if they are never bred. They grow wool every 6 days.'),
+            entity('tfc:sheep', 'A sheep.', 'Sheep', scale=1),
+            text('$(thing)Sheep$() spawn in drier climates, $(l:the_world/climate#temperature)temperature$() between 0 and 35°C, and 70 and 300mm of $(l:the_world/climate#rainfall)rainfall$(). They are $(thing)wooly animals$(). They eat $(item)grains$(). They have 1-2 children, are pregnant for 32 days, and reach adulthood in 56 days. They can have children 6 times, if they are never sheared, or be sheared 60 times, if they are never bred. They grow wool every 9 days.'),
```

Comment:
Rather than `$(thing)wooly animals$()`, link this to the wooly animal anchor, and same with the dairy animals?